std/net/curl.d

```diff
@@ -4883,285 +4778,397 @@ private struct Pool(Data)
     }
 }
 
-// Shared function for reading incoming chunks of data and
-// sending the to a parent thread
-private static size_t _receiveAsyncChunks(ubyte[] data, ref ubyte[] outdata,
-                                          Pool!(ubyte[]) freeBuffers,
-                                          ref ubyte[] buffer, Tid fromTid,
-                                          ref bool aborted)
+// Lazily-instantiated namespace to avoid importing std.concurrency until needed.
+private struct _async()
 {
-    import std.concurrency : receive, send, thisTid;
+static:
+    // @@@@BUG 15831@@@@
+    // this should be inside byLineAsync
+    // Range that reads one chunk at a time asynchronously.
+    private static struct ChunkInputRange
```

Comment:
`static` is redundant here.

Reply:
Err actually there were two more non-templated functions.